docs/usage-as-end-user.md

```diff
@@ -106,6 +112,9 @@ loadBalancerClasses:
 cloudControllerManager:
   featureGates:
     CustomResourceValidation: true
+#csi
```

Comment:
this needs to be updated right ? And can we also update the example/{infrastructure,controlplane} files ?

Reply:
Thanks, missed that.